src/browser.ts

```diff
@@ -1,3 +1,7 @@
 export function getUserAgent(): string {
-  return navigator.userAgent;
+  try {
+    return navigator.userAgent;
+  } catch (e) {
+    return 'unknown';
```

Comment:
I think we should return something that is less ambiguous. Returning `unknown` would result in `User-Agent` header values such as `octokit-graphql.js/1.2.3 unknown`. What do you think about `octokit-graphql.js/1.2.3 <navigator unknown>`

```suggestion
    return "<navigator unknown>"
```

Reply:
Makes a ton of sense, let's make that change 👍